packages/contracts-bedrock/scripts/deploy/DeployOPCM.s.sol

```diff
@@ -311,3 +333,102 @@ contract DeployOPCM is Script {
         doo_ = DeployOPCMOutput(DeployUtils.toIOAddress(msg.sender, "optimism.DeployOPCMOutput"));
     }
 }
+
+
+// Create a self contained script for deploying an OPPrestateUpdater with mostly dummy args as
+//
+contract DeployOPCMRunner is Script {
```

Comment:
This was my way of deploying the OPPrestateUpdater without having an op-deployer command for it. The decision has been made to skip op-deployer support, so we will need to be able to deploy this with a forge script. Therefore, we probably do want something like this in the final Deploy script.

Theere should also be a test similar to DeployImplementations.t.sol.